base-system/tools/installer/install.sh

```diff
@@ -160,6 +185,12 @@ cp --verbose -rf "$ISO_DIR/EFI/" "$SYSTEM_MNT"
 cp --verbose -rf "$ISO_DIR/checksums" "$SYSTEM_MNT"
 cp --verbose -rf "$SERVER_CONFIG" "$SYSTEM_MNT/huronOS/data/configs/sync-server.conf"
 
+# If a password was passed, update the password
+if [ -n "$NEW_PASSWORD" ]; then
+  print_step "[7.5/11] Setting new password: $NEW_PASSWORD"
+  ./change-password.sh "$NEW_PASSWORD" "$SYSTEM_MNT" || exit 1
+fi
```

Comment:
I think that it would be more helpful to show it at the end of the installer, I'll mention which is the password that was set on that instalation, on my way to update it.